substrate/frame/revive/src/exec/tests.rs

```diff
@@ -1142,6 +1142,13 @@ fn instantiation_work_with_success_output() {
 				ContractInfo::<Test>::load_code_hash(&instantiated_contract_id).unwrap(),
 				dummy_ch
 			);
+			assert_eq!(
+				&events(),
+				&[Event::Instantiated {
+					deployer: ALICE_ADDR,
+					contract: instantiated_contract_address
+				}]
+			);
```

Comment:
This is the only test that was effected.

I've checked there's also a test that already checks the case when the instantiation call succeeds but the contract is reverted - In that case there's already an assertion that checks that no events should be emitted. See `instantiation_fails_with_failing_output`.

All these tests were done previously, but updated the event assertions when #7164 removed the events. The only one that needed to be updated is this one.